reference/configuration/framework.rst

```diff
@@ -1,5 +1,5 @@
 .. index::
-    single: Configuration reference; Framework
+single: Configuration reference; Framework
```

Comment:
This needs to be removed Otherwise this line is not treated as an argument for the index.